website/blog/2023-01-02-aluminium.md

```diff
@@ -0,0 +1,769 @@
+
+In the previous version of Metals additional library dependency suggestions were
+add in Ammonite scripts and worksheets. This version expands the number of
+places where you can use the feature:
+
+- ScalaCLI scripts and files when writing `//> using lib ...`
+- When writing `ivy"dependency..."` in build.sc file
+- Inside sbt files when adding dependencies.
+
+This feature allows you to explore the available libraries and their versions
+without exiting the comforts of your editor.
+
+Thanks [jkciesluk](https://github.com/jkciesluk) and
+[LaurenceWarne](https://github.com/LaurenceWarne) for all the great work!
+
+## Actionable diagnostics
+
+Actionable diagnostics is a new idea within the Scala tooling ecosystem that
+focuses on providing users with a potential fix for any errors or warnings that
+may show up. This reduces the amount of time users will spend on fixing their
+code and help them be more productive in daily development.
+
+The first experiment within that area was done by
+[lwronski](https://github.com/lwronski) in
+[ScalaCLI](https://scala-cli.virtuslab.org/). When using the newest ScalaCLI you
+will see warning whenever a library has a newer version available and have a
+quick fix suggested via code action.
+
+![actionable](https://i.imgur.com/3D9WcQb.gif) These warnings can be disabled
+within ScalaCLi using:
+
+```
+scala-cli config actions false
+```
+
+There is an ongoing work on adding more actionable diagnostics in the Scala 3
+compiler.
+
+## Expose Test Cases as Separate Lenses
+
+Up until the current release it was only possible to run a single test case
+within Visual Studio Code which uses an additional Test Explorer API that is not
+available for any other editors. However, thanks to
+([LaurenceWarne](https://github.com/LaurenceWarne)) Metals will now show lenses
+to run a single test case in case of all other editors.
+
+More details can be found in the
+[PR \#4569](https://github.com/scalameta/metals/pull/4569).
+
+## Add support for type definition
+
+Users were able for a long time to navigate to the definition of a particular
+symbol, which can be a method, object, class etc. After that they could
+sometimes also find the definition of the type of that symbol. However, it's not
+always easy and convenient to do that. From now one Metals also allows users to
+use the `go to type definition` feature within all the supported editors. User
+will be able to go directly to the type of a particular symbol.
+
+![type-def](https://i.imgur.com/t77kB1S.gif)
+
+This feature might work a bit differently depending on the symbol you are
+checking:
+
+- for method it will show the location of the return type
+- for any values, vars or parameters it will show the type of that symbol
+- for classes, object or types it will only find that particular type, which
+  will work the same as `go to definition`
+
+## Run main classes in terminal when using code lenses
+
+More to come, tests not yet
```

Comment:
TODO just make sure to finish this, but I'd also recommend being careful with the wording here. For example when I read this at first I was thinking `runIntTerminal` which is part of DAP, and something we don't support. So I think we should clarify that this just means we're directly running without starting a debug server, not to be confused with the `runInTerminal` from DAP.

Reply:
Ok, I think I maanged to get the wording ok, let me know!

Reply:
I wonder how many people who read release notes have any idea what DAP is ;)

Reply:
No many I think :sweat_smile: that's why I avoided using it there.